Assets/Python/Contrib/Sevopedia/SevoPediaBuilding.py

```diff
@@ -289,7 +289,7 @@ def getBuildingSortedList(self, iBuildingType):
 		numInfos = 0
 		for iBuilding in range(gc.getNumBuildingInfos()):
 			if gc.getBuildingInfo(iBuilding).isGraphicalOnly() or iBuilding >= con.iNumBuildings: continue
-			if iBuilding in [con.iBabylonGarden, con.iZuluIkhanda, con.iChinesePavillion, con.iIncanTerrace, con.iNativeAmericaTotem]: continue
+			if iBuilding in []: continue

```

Comment:
I suppose that's covered by bGraphicalOnly now? If so, just remove the entire line, this is just dead code.